apps/meteor/app/oauth2-server-config/server/admin/methods/deleteOAuthApp.ts

```diff
@@ -1,11 +1,14 @@
 import { Meteor } from 'meteor/meteor';
 import { OAuthApps } from '@rocket.chat/models';
 
-import { hasPermission } from '../../../../authorization';
+import { hasPermission } from '../../../../authorization/server';
+import { methodDeprecationLogger } from '../../../../lib/server/lib/deprecationWarningLogger';
 
 Meteor.methods({
 	async deleteOAuthApp(applicationId) {
-		if (!hasPermission(this.userId, 'manage-oauth-apps')) {
+		methodDeprecationLogger.warn('deleteOAuthApp will be deprecated in future versions of Rocket.Chat');
```

Comment:
This is not a correct statement, the correct would be `deleteOAuthApp is deprecated and will be in future versions of Rocket.Chat`..

But recently I was thinking that we should give additional information on what should be used instead, something like:
```suggestion
		methodDeprecationLogger.warn('deleteOAuthApp is deprecated and will be in future versions of Rocket.Chat. Use the REST endpoint /v1/oauth-apps.delete instead.');
```